src/github/deployment.ts

```diff
@@ -80,3 +98,36 @@ export const processDeployment = async (
 		gitHubAppId
 	);
 };
+
+const persistentSuccessDeploymentStatusToDynamoDB = async (
+	jiraHost: string,
+	gitHubBaseUrl: string,
+	gitHubInstallationId: number,
+	gitHubAppId: number | undefined,
+	webhookPayload: WebhookPayloadDeploymentStatus,
+	logger: Logger
+) => {
+
+	const gitHubProduct = getCloudOrServerFromGitHubAppId(gitHubAppId);
+	const tags = { gitHubProduct };
+	const info = { jiraHost };
+
+	try {
+		statsd.increment(metricDeploymentPersistent.toCreate, tags, info);
+		await saveDeploymentInfo({
+			gitHubBaseUrl,
+			gitHubInstallationId,
+			repositoryId: webhookPayload.repository.id,
+			commitSha: webhookPayload.deployment.sha,
+			description: webhookPayload.deployment.description || "",
```

Comment:
do we really need description? It might contain UGC/PII...

Reply:
removed. good point.